src/types/backend.d.ts

```diff
@@ -1,10 +1,12 @@
 export interface User {
-  firstName: string
-  lastName: string
-  profilePicture: string
+  first_name: string
+  last_name: string
+  profile_picture: string
   email: string
   id: string
   avatar: string
   accessToken?: string
+  access?: string
+  refresh?: string
```

Comment:
Should this be included in the user interface? I'm not convinced that this belongs here, especially after seeing that the token is stored separately in AsyncStorage.